examples/make_tracing_sc/sc_main.cpp

```diff
@@ -94,7 +96,7 @@ int sc_main(int argc, char* argv[]) {
         tfp = new VerilatedVcdSc;
         top->trace(tfp, 99);  // Trace 99 levels of hierarchy
         Verilated::mkdir("logs");
-        tfp->open("logs/vlt_dump.vcd");
+        tfp->open("logs/vlt_dump.fst");
```

Comment:
I believe the default makefile in this example makes a VCD, so this should be reverted.